### DISCUSSION TITLE
This page is taking too long to load. (Unicorn)

### DISCUSSION THREAD

**Discussion by Lychas** [community/community#38724] [Projects and Issues]
### Select Topic Area

Bug

### Body

Whenever I try to access https://github.com/apps/aws-amplify-eu-central-1/installations/new or  https://github.com/apps/aws-amplify-eu-central-1 , it waits for like half a minute and then shows me this message, I have no idea why is it happening and everything in GitHub is working well, nothing seems wrong with the account. 
![image](https://user-images.githubusercontent.com/95055527/200951443-84459209-d082-4f08-ae33-59632b92da30.png)

**Comment by ossanna16** [comment]
Hi there @Lychas and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
![image](https://user-images.githubusercontent.com/47092464/201529201-22203de4-545a-43bd-a6c2-59b083bd44c5.png)

in the case of the Gist, that is very large one, as you can see at the scroll button, it just loaded for some seconds on my end, but this computer and the connection is very fast, so for lower specs, that will really take time to load

as for the other two

![image](https://user-images.githubusercontent.com/47092464/201529114-2fa605c7-cad8-42ab-b83c-e09168c31f94.png)

there is no loading at all

![image](https://user-images.githubusercontent.com/47092464/201529132-eedb2cc1-9c2b-4c56-883e-2665433ddf25.png)

so if you have either extensions installed, it can affect the way GitHub Web functions and as I've mentioned, on the end user, slow computer, poor connection, low RAM etc.,
